packages/node/src/fetch.ts

```diff
@@ -7,13 +12,28 @@ export const Fetch: Plugin = (_options, exporter) => {
   global.fetch = async (...args) => {
     const id = generateId();
     const startTs = performance.now();
+    let response: Response | undefined = undefined;
 
     // export the initial request data
     const reqEvent = getEventFromFetchRequest(id, ...args);
     exporter.send(reqEvent);
 
+    // if the args contain a signal, listen for abort events
+    const signal = args[1]?.signal;
+    if (signal) {
+      signal.addEventListener('abort', () => {
+        // if the request has already been resolved, we don't need to do anything
+        if (response) return;
```

Comment:
❓ Would this ever have a value if the `abort` event was triggered?

🤔 I'm also just trying to work out whether the value of `response` would be the value from line 36, given that the closure which this exists in will likely have gone out of scope.

Reply:
I have a small example here, where I am aborting the request after 1 second (after it has resolved). It seems to have the response when I log from within the abort event:

https://github.com/FormidableLabs/envy/assets/9557798/245f6c27-33b7-4001-a59f-47d8070fe27e

Reply:
Fair enough.  Lemme 👍 this